src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -4,22 +4,22 @@ import { Link } from 'react-router';
 const Breadcrumbs = ({ query, type, title, url }) => {
   let currentPageText = 'Research Catalog';
   let crumbs = (
-    <span>
-      <Link to="https://nypl.org">Home</Link> &gt;&nbsp;
-      <Link to="https://nypl.org/research">Research</Link> &gt;&nbsp;
-      <span className="currentPage">{currentPageText}</span>
-    </span>
+    <ol className="nypl-breadcrumbs">
```

Comment:
I rather not have `<ol className="nypl-breadcrumbs">` repeated multiple times in this file but I see why you did that. Perhaps later we can update it to make it render it better.